tests/nlu/test_rule_based_annotator.py

```diff
@@ -38,16 +39,22 @@
 }
 
 
-def process_value(text: str) -> str:
-    """Returns a process value fixture."""
+def mock_process_value(text: str) -> str:
```

Comment:
shouldn't this be marked @pytest.fixture if it is a fixture?

Reply:
Fixed docstring